features/steps/givens/attributes.py

```diff
@@ -7,9 +7,14 @@
 
 @given("{attribute} = {value}")
```

Comment:
```suggestion
@given("{attribute} = {value}")
@given("{attribute} must be of value {value}")
```
Shall we add 'must be of value' for this? 

Allowes for the given statement (as an example). 
`"And PredefinedType must be of value 'POSITION' or 'STATION'"`

Perhaps this will it will a bit easier to implement for others.

Reply:
Let's save this for later